salt/orchestrate/operations/backups.sls

```diff
@@ -6,6 +6,7 @@
     'public1-{}'.format(VPC_RESOURCE_SUFFIX), 'public2-{}'.format(VPC_RESOURCE_SUFFIX), 'public3-{}'.format(VPC_RESOURCE_SUFFIX)])['subnets'] %}
 {% do subnet_ids.append('{0}'.format(subnet['id'])) %}
 {% endfor %}
+{% set slack_api_token = salt.vault.read('secret-operations/global/slack/slack_api_token.data.value') %}
```

Comment:
`.data.value` needs to be oustside the parentheses for the `.read` function call.